webapp/channels/src/components/admin_console/system_users/system_users_dropdown/system_users_dropdown.tsx

```diff
@@ -737,7 +737,7 @@ export default class SystemUsersDropdown extends React.PureComponent<Props, Stat
                         </SystemPermissionGate>
                         <SystemPermissionGate permissions={[Permissions.SYSCONSOLE_WRITE_USERMANAGEMENT_GROUPS]}>
                             <Menu.ItemAction
-                                show={user.auth_service === Constants.LDAP_SERVICE}
+                                show={user.auth_service === Constants.LDAP_SERVICE || (user.auth_service === Constants.SAML_SERVICE && config.SamlSettings?.EnableSyncWithLdap)}
```

Comment:
If LDAP sync is enabled, will this show for non-LDAP user, which authenticate using SAML?

Is there a way to check where a user originated from, i.e. if it's created via Email/Password or by the LDAP server?

Reply:
Unfortunately I don't think we have a way to distinguish this. Is it even possible to have non-LDAP SAML users if that sync is enabled?

Reply:
You are correct. All SAML users will have their attributes synced with LDAP if `EnableSyncWithLdap` is enabled.